charts/external-services/values.yaml

```
@@ -9,14 +9,23 @@ externalSecret:
   provider: ""
 
 gateway:
-  enabled: false
+  enabled: true
   trustedIPs:
   - 10.0.0.0/8
   - 172.0.0.0/8
   ports:
     grpc:
       kind: GRPCRoute
       port: 31238
+    http:
+      kind: HTTPRoute
+      port: 80
+      transport:
+        respondingTimeouts:
+          readTimeout: 0
+    https:
+      kind: HTTPRoute
+      port: 443
```

```suggestion
  enabled: false
  trustedIPs:
  - 10.0.0.0/8
  - 172.0.0.0/8
  ports:
    grpc:
      kind: GRPCRoute
      port: 31238
```
enabled is not required here (sorry for ambiguous naming)
http and https are handled by default traefik values so not needed here (web, websecure, respectively)